src/main/java/ladder/io/ConsoleOutputHandler.java

```diff
@@ -1,39 +1,40 @@
 package ladder.io;
 
+import ladder.*;
+
 import java.util.List;
-import ladder.Ladder;
-import ladder.Line;
-import ladder.Lines;
-import ladder.Points;
 
 public class ConsoleOutputHandler implements OutputHandler {
 
     @Override
     public void printLadder(Ladder ladder) {
-        final Lines lines = ladder.getLines();
-        for (Line line : lines.getLines()) {
+        final List<Line> lines = ladder.getLines();
+        for (Line line : lines) {
```

Comment:
저는 지금처럼 변수 네이밍이 어려운 경우에는 변수로 객체를 담아서 처리하기 보다는 바로 이용하기도 합니다.
```suggestion
        for (Line line : ladder.getLines()) {
```